runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItem.java

```diff
@@ -43,7 +45,14 @@
 	private int gePrice;
 	private int offset;
 	private boolean tradeable;
-	private boolean isMine;
+	private boolean isPvpLoot;
```

Comment:
should this be just Enum or something? or boolean? Because it can be either pvp or pvm, not both.

Reply:
Could do an enum I guess but there are 3 separate things: pvm, pvp, neither.

Reply:
Yeah, I realized after I commented. Then maybe 3 state enum (PvP, PvM, Unknown)?